website/pages/docs/plugins/sources/datadog/configuration.mdx

```diff
@@ -16,6 +16,10 @@ This is the (nested) spec used by the Datadog source plugin.
 
   Specify which accounts to sync data from.
 
+- `site` (`string`, optional, default: "")
```

Comment:
Is setting an empty string (using the default) the same as setting it to `datadoghq.com`? Maybe we should clarify that if that's the case

Reply:
Yes, it falls back on [this](https://github.com/DataDog/datadog-api-client-go/blob/master/api/datadog/configuration.go#L126) configuration if there is not `site` override in the context.